QgisModelBaker/utils/gui_utils.py

```diff
@@ -115,45 +115,57 @@ class DropMode(Enum):
 ]
 
 # style
-ORANGE = "#D1C28E"
+ORANGE = "#D1C28E"  # Not used
 GREEN = "#A1DE9B"
 BLUE = "#9BCADE"
-PURPLE = "#B18BC9"
-RED = "#EBB3A4"
+PURPLE = "#B18BC9"  # Not used
+RED = "#ff0000" if QgsApplication.themeName() == "Night Mapping" else "#EBB3A4"
```

Comment:
Can we have softer colors here. Similar to the bright theme. While there we had more white in it (pastel colors) maybe with the dark more grayish colors could work...

Reply:
The orange color is also used by the palette, and actually used in all toolbars for intermediate steps.

Would this red work for you?

![image](https://github.com/opengisch/QgisModelBaker/assets/652785/1f3122be-2eec-4556-8565-0d6e9a51319c)

What about this green color? (Compare it with the selected dataset color)

![image](https://github.com/opengisch/QgisModelBaker/assets/652785/6dcd2858-d55f-4d27-a141-e16347ab0c27)

Reply:
Yeah this is nice. The other made me leave through the emergency escape everytime when the validation failed ;-)

Orange and green is fine. Thanks.